deploy/docs/V1_Migration_doc.md

```diff
@@ -0,0 +1,177 @@
+```
+#### 2: Run upgrade script
+
+For Helm users, the only breaking changes are the renamed config parameters.
+For users who use a `values.yaml` file, we will provide a script that customers can run to convert their existing `values.yaml` files into ones that are compatible with the major release. 
+
+The upgrade script is present in the path : `sumologic-kubernetes-collection/deploy/helm/sumologic/`
+You will either need to clone the repo or `curl` the upgrade script. 
+Run the upgrade script with the below command.
+```bash
+./upgrade-1.0.0.sh values.yaml
+```
+At this point, users can then run: 
+```bash
+helm upgrade collection sumologic/sumologic --reuse-values --version=1.0.0 -f new_values.yaml
+```
+For users who installed without a `values.yaml` file, if they had set any of the above config parameters they will need to re-set those parameters during their upgrade. For example, if the customer had originally run
+```bash
+helm install sumologic/sumologic --name collection --namespace sumologic --set sumologic.accessId=<SUMO_ACCESS_ID> ... --set sumologic.numThreads=16
+```
+Then their upgrade command will look like
+```bash
+helm upgrade collection sumologic/sumologic --reuse-values --version=1.0.0 --set fluentd.buffer.numThreads=16
+```
+## Non-Helm Users
+### Breaking Changes
+- The use of environment variables to set configs has been removed to avoid the extra layer of indirection and confusion. Instead, configs will be set directly within the Fluentd pipeline.
+- `kubernetesMeta` and `kubernetesMetaReduce` have been removed from `logs.kubernetes.sumologic.filter.conf` of the Fluentd pipeline for the same reason as above (Helm users)
+- Similarly `addStream` and `addTime` (default values were `true`) have been removed from `logs.kubernetes.sumologic.filter.conf` of the Fluentd pipeline; the default behavior will remain the same. To preserve the behavior of `addStream = false` or `addTime = false`, you can add: 
+```yaml
+<filter containers.**>
+  @type record_modifier
+  remove_keys stream,time
+</filter>
+```
+above the output plugin section [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/release-v1.0.0/deploy/kubernetes/fluentd-sumologic.yaml.tmpl#L251)
+- The Fluentd deployments have been changed to **statefulsets** to support the use of **persistent volumes**. This will allow better buffering behavior. They also now include `“fluentd”` in their names. This is a breaking change for non-Helm users as the deployments will not be cleaned up upon upgrade, leading to duplicate events (logs and metrics will not experience data duplication).
+- The unified Fluentd `statefulsets` have been split into set of two different Fluentd's, one for `logs` and the other one for `metrics`.
+- We now support the collection of renamed metrics (for Kubernetes version `1.17+`).
+### How to upgrade for Non-helm Users
+#### 1. Tear down existing fluentd, prometheus, fluentbit resources
```

Comment:
In general I think we should capitalize Fluentd Prometheus and FluentBit in our docs